### ISSUE TITLE
model save error

### ISSUE THREAD

**Issue by KavitaHoude** [sovit-123/fasterrcnn-pytorch-training-pipeline#122]
I am getting the following errors when trying to train the model on custom dataset. This error is getting at last epoch. Maybe it is model save error. Please give suggestions to solve these errors.

**Comment by KavitaHoude** [comment]
hello Sir,
this is the command
!python train.py --model fasterrcnn_mobilenetv3_large_fpn --data
data_configs/custom_data.yaml --epochs 10 --name
fasterrcnn_mobilenetv3_large_fpn_noaug_40e --seed 42

**Comment by sovit-123** [comment]
Okay. If you are training on Colab and trying save on Google Drive, please use the `--project-dir` argument instead of the `--name` argument for saving the project.

**Comment by KavitaHoude** [comment]
> Okay. If you are training on Colab and trying save on Google Drive, please use the `--project-dir` argument instead of the `--name` argument for saving the project.

not worked. same error again

**Comment by sovit-123** [comment]
Can you please let me know where the code files are? Is it getting cloned to colab or is it somewhere on the Google Drive? It may not work If it is on Google Drive.

**Comment by KavitaHoude** [comment]
> Can you please let me know where the code files are? Is it getting cloned to colab or is it somewhere on the Google Drive? It may not work If it is on Google Drive.

its cloned to google drive by using the command
!git clone https://github.com/sovit-123/fasterrcnn-pytorch-training-pipeline.git

**Comment by sovit-123** [comment]
Most probably it won't run from Google Drive. Please try to clone to the colab drive directly and run it.
